### PR TITLE
The first Heading

### PR DESCRIPTION
**Pull request by AdrianNQ99**

### CODE REVIEW

**Review by PilarGallegoS — CHANGES_REQUESTED** [review]
cambio clase

**Review by PilarGallegoS — DISMISSED** [review]
cambios ok

File: website/templates/first_heading.html
Comment (ScarzoJ) [inline website/templates/first_heading.html:1]: Format whole file

File: website/templates/first_heading.html [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+            quam minus quis quas quae incidunt expedita molestiae enim cum hic, ab cupiditate magni fuga inventore autem
+            nam?</p>
+    </div>
+    <div class="col-4">
+        <img src="../website/assets/img/1.jpg" class="img-fluid rounded-circle" alt="...">
+    </div>
+</div>
+</div>
+<hr>
 <p>Hello, I'm the first_heading.html</p>
```
Comment (ScarzoJ) [inline website/templates/first_heading.html]: Remove unused line from example

**Review by ScarzoJ — CHANGES_REQUESTED** [review]

**Pull request closed without merging**